Descriptive logs with docLinks for cluster shard limit exceeded (#132072)
elastic/kibana: docs/setup/upgrade/resolving-migration-failures.asciidoc

  }
}
--------------------------------------------

[float]
[[cluster-shard-limit-exceeded]]
==== Elasticsearch cluster shard limit exceeded

> When upgrading, {kib} creates new indices requiring a small number of new shards. If the amount of open {es} shards approaches or exceeds the Elasticsearch `cluster.max_shards_per_node` setting, Kibana is unable to complete the upgrade. Ensure that Kibana is able to add at least 10 more shards by removing indices to clear up resources, or increasing the `cluster.max_shards_per_node` setting.

Reviewer: ```suggestion
When upgrading, {kib} creates new indices requiring a small number of new shards. If the amount of open {es} shards approaches or exceeds the Elasticsearch `cluster.max_shards_per_node` setting, {kib} is unable to complete the upgrade. Ensure that {kib} is able to add at least 10 more shards by removing indices to clear up resources, or by increasing the `cluster.max_shards_per_node` setting.
```

Reviewer 2: also:
```suggestion
When upgrading, {kib} creates new indices requiring a small number of new shards. If the amount of open {es} shards approaches or exceeds the {es} `cluster.max_shards_per_node` setting, Kibana is unable to complete the upgrade. Ensure that Kibana is able to add at least 10 more shards by removing indices to clear up resources, or increasing the `cluster.max_shards_per_node` setting.
```

> Refer to the {es} documentation for more information {ref}/allocation-total-shards.

Reviewer: This should fix the broken link:

```suggestion
For more information, refer to the documentation on {ref}/allocation-total-shards.html[total shards per node].
```
